tony-history-server/gradlew

```diff
@@ -0,0 +1,172 @@
+#!/usr/bin/env sh
```

Comment:
You shouldn't need to check in another gradlew and gradle/ directory under tony-history-server. You should be able to build using the gradlew + gradle/ in the root of the TonY project. E.g.:

```
# from the root of TonY repo
./gradlew :tony-history-server:build
```